web-local/src/routes/(viz)/+layout.svelte

```diff
@@ -29,9 +33,19 @@
     return map;
   }, new Map<string, PathOption>());
 
-  $: pathParts = [dashboardOptions];
+  $: huh = <PathOption>{
```

Comment:
`huh` is from Brian's dev mode, right...?

Reply:
🫠